packages/containers/src/EditableText/EditableText.container.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EditableText as Component } from '@talend/react-components';
+import Immutable from 'immutable';
+import omit from 'lodash/omit';
+import { cmfConnect } from '@talend/react-cmf';
+
+export const DISPLAY_NAME = 'Container(EditableText)';
+export const DEFAULT_STATE = new Immutable.Map({
+	editMode: false,
+});
+
+class EditableText extends React.Component {
+	static displayName = DISPLAY_NAME;
+
+	static propTypes = {
+		...cmfConnect.propTypes,
+		actionCreatorCancel: PropTypes.string,
+		actionCreatorEdit: PropTypes.string,
+		actionCreatorSubmit: PropTypes.string,
+		actionCreatorChange: PropTypes.string,
+		onCancel: PropTypes.func,
+		onEdit: PropTypes.func,
+		onSubmit: PropTypes.func,
+		onChange: PropTypes.func,
+		text: PropTypes.string,
+	};
+
+	static contextTypes = {
+		registry: PropTypes.object,
+		store: PropTypes.object,
+	};
+
+	constructor(props) {
+		super(props);
+		this.onCancel = this.onCancel.bind(this);
+		this.onEdit = this.onEdit.bind(this);
+		this.onSubmit = this.onSubmit.bind(this);
+		this.onChange = this.onChange.bind(this);
+	}
+
+	onSubmit(event, data) {
+		this.props.setState({ editMode: false });
+		if (this.props.onSubmit) {
+			this.props.onSubmit(event, data);
+		}
+		if (this.props.actionCreatorSubmit) {
+			this.props.dispatchActionCreator(this.props.actionCreatorSubmit, event, {
+				props: this.props,
+				data,
+			});
+		}
```

Comment:
you need this ? `onSubmitActionCreator` should work (become a onSubmit)

Reply:
As discussed, this will cause a lot of breaking changes. Maybe we could do that in a second time...